pages/common/tmux.md

```diff
@@ -2,8 +2,7 @@
 
 > Tmux is a terminal multiplexer. It lets you switch easily between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal.
 > More information: <https://github.com/tmux/tmux>.
-> By default, your tmux prefix is Ctrl-b.  In the examples below, it'll mention {{prefix}}.
-> If you haven't changed your prefix, use Ctrl-b.  If you have, use your prefix.
+> By default, your tmux prefix is Ctrl-b.  In the examples below, it'll mention prefix. If you haven't changed your prefix, use Ctrl-b.  If you have, use your prefix.
```

Comment:
It should be Ctrl-B or Ctrl+B. What do others think?

Reply:
Are you referring to how I talk about the keybinding?

Ex. Ctrl-b -> Ctrl-B

Reply:
Yes

Reply:
I'll switch to Ctrl-B.  If anyone disagrees, I can make a further change later.